packages/button/src/css/index.js

```diff
@@ -95,53 +86,126 @@ export default {
     padding: `${layout.spacingXXSmall} ${layout.spacingMedium}`,
     height: '48px'
   },
-  [`.psds-button--appearance-${vars.appearances.secondary}`]: {
+  [`.psds-button--appearance-${vars.appearances.primary}.psds-theme--${themeDefaultName}`]: {
+    color: colorsTextIcon.highOnDark,
+    background: colorsPrimaryAction.background,
+    '&:not([disabled]):hover': {
+      background: colorsBlue[5],
+      cursor: 'pointer'
+    },
+    '&:not([disabled]):focus': {
+      background: colorsBlue[5],
+      cursor: 'pointer'
+    },
+    '&:not([disabled]):active': {
+      background: colorsBlue[7],
+    }
+  },
+  [`.psds-button--appearance-${vars.appearances.primary}.psds-theme--${themeNames.light}`]: {
+    color: colorsTextIcon.highOnDark,
+    background: colorsPrimaryAction.background,
+    '&:not([disabled]):hover': {
+      background: colorsBlue[7],
+      cursor: 'pointer'
+    },
+    '&:not([disabled]):focus': {
+      background: colorsBlue[7],
+      cursor: 'pointer'
+    },
+    '&:not([disabled]):active': {
+      background: colorsBlue[8],
+    }
```

Comment:
should these colors be part of the primary action palette

Reply:
All states have AA or better. The only one that didn't was the primary hover state on dark.

@danethurber I did as much as I could to try to adjust the styling of the buttons inside banners to improve contrast, but was unable to get it to build so if you want to take a look and maybe we could jump on a call to review it.